src/components/inputs/commons/InputDescription.tsx

```diff
@@ -13,5 +13,5 @@ export const InputDescription = styled(Text).attrs({
 })`
 	line-height: 1.5;
 	padding-top: 0.25rem;
-	min-height: calc(${({ theme, size }): string => theme.sizes.font[size]} * 1.5);
+	min-height: calc(${({ theme, size }): string => theme.sizes.font[size ?? 'medium']} * 1.5);
```

Comment:
The size is always valued from the attrs, and it is set to extrasmall. We need to understand how to make the styled component understand it (an alternative solution could be export a component which inside uses a styled component)